authenticate/authenticate.cabal

```diff
@@ -19,7 +19,7 @@ flag network-uri
 
 library
     build-depends:   base                          >= 4.10     && < 5
-                   , aeson                         >= 0.5
+                   , aeson      >= 0.5 && < 1.6 || >= 2.0 && < 2.1
```

Comment:
I'm not interested in adding upper bounds here, please remove.

Reply:
I removed upper bounds.